src/streamlink_cli/main.py

```diff
@@ -1006,7 +1006,7 @@ def check_version(force=False):
                             "available!".format(latest_version))
         cache.set("version_info_printed", True, (60 * 60 * 6))
     elif force:
-        console.logger.info("Your Streamlink version ({0}) is up to date!",
+        console.logger.info("Your Streamlink version ({0}) is up to date.",
```

Comment:
Why? This could potentially break 3rd party applications. And since log messages are all over the place and don't follow a certain pattern, having an exclamation mark here instead of a full stop doesn't improve the log messages as a whole.

Having a globally unified log format would be nice, but I guess that would require a major version bump, wouldn't it?